tests/unit/xoopsLib/Xoops/Core/Cache/AccessTest.php

```diff
@@ -1,8 +1,8 @@
 <?php
-namespace Xoops\Core\Cache;
```

Comment:
Why are we removing these from namespaces? Most projects seem to use namespaced tests. We have already had conflicts with the same class name used in several test classes. Using the natural namespace seems like a good idea, and standard practice.

Are you having issues when using them?